1_1_vulns/it/LLM10_ModelTheft.md

```diff
@@ -1,49 +1,49 @@
+8. Implementare un addestramento avversariale mirato alla resilienza, sia per aiutare a rilevare query di estrazione, e rafforzare le misure di sicurezza fisica.
+9. Implementare un framework di marcatura modale e|o temporale (watermarking) nelle fasi di integrazione (input) e monitoraggio (output, trasformazione), parte del ciclo di vita di un LLM.
 
-### Example Attack Scenarios
+### Esempi di Scenario di Attacco
 
-1. An attacker exploits a vulnerability in a company's infrastructure to gain unauthorized access to their LLM model repository. The attacker proceeds to exfiltrate valuable LLM models and uses them to launch a competing language processing service or extract sensitive information, causing significant financial harm to the original company.
-2. A disgruntled employee leaks model or related artifacts. The public exposure of this scenario increases knowledge to attackers for gray box adversarial attacks or alternatively directly steal the available property.
-3. An attacker queries the API with carefully selected inputs and collects sufficient number of outputs to create a shadow model.
-4. A security control failure is present within the supply-chain and leads to data leaks of proprietary model information.
-5. A malicious attacker bypasses input filtering techniques and preambles of the LLM to perform a side-channel attack and retrieve model information to a remote controlled resource under their control.
+1. Un attaccante abusa una vulnerabilità nell'infrastruttura di un'azienda per accedere senza autorizzazione al loro repository di modelli LLM. L'attaccante procede quindi all'esfiltrazione di modelli LLM di valore e li utilizza per lanciare un servizio concorrente di elaborazione del linguaggio o estrarre forzatamente informazioni sensibili, causando gravi danni finanziari all'azienda proprietaria del modello.
+2. Un dipendente insoddisfatto divulga modelli o artefatti correlati. L'esposizione pubblica che questo scenario rappresenta, aumenta la conoscenza di attaccanti ulteriori, consentendo attacchi avversariali di tipo "gray box" (conoscenza parziale) o, in alternativa, consente di rubare direttamente la proprietà intellettuale esposta.
+3. Un attaccante interroga l'API con input accuratamente selezionati e raccoglie un numero sufficiente di output per creare un modello ombra (shadow).
+4. Una mancanza, nei controlli di sicurezza della catena di fornitura (supply chain) porta a perdite di dati relativi a informazioni proprietarie del|sul modello.
+5. Un attaccante malintenzionato elude le tecniche di filtraggio degli input e negli stadi iniziali dell'LLM, per eseguire un attacco indiretto|laterale e recuperare informazioni sul modello verso una risorsa remota, sotto il suo controllo.
```

Comment:
Not 100% sure what "preambles of the LLM" means here, but I would interpret it as the initial instructions given to the LLM (e.g. the part of the prompt preceding the user-provided text). Would that make sense?

```suggestion
5. Un attaccante malintenzionato elude le tecniche di filtraggio degli input e le istruzioni iniziali dell'LLM, per eseguire un attacco indiretto/laterale e recuperare informazioni sul modello caricandole su una risorsa remota sotto il suo controllo.
```